modules/AlphaWalletFoundation/AlphaWalletFoundation/EtherClient/EIP712TypedData.swift

```diff
@@ -35,6 +35,34 @@ public struct EIP712TypedData: Codable {
         }
     }
 
+    public var server: RPCServer? {
+        switch domain {
+        case .object(let dictionary):
+            switch dictionary["chainId"] {
+            case .string(let str):
+                if let server = eip155URLCoder.decodeRPC(from: str) {
+                    return server
+                } else if let value = Int(str, radix: 10) {
+                    return RPCServer(chainID: value)
+                } else if let value = Int(str) {
```

Comment:
Isn't this the same as with `radix: 10` above?

If we keep just 1, let's keep the `radix: 10` variation because sometimes chain IDs are expected to be base 16, so it's good to be explicit in our handling